### ISSUE TITLE
 doubts about adjustments -- By Google Translate

### ISSUE THREAD

**Issue by dbmb91** [gh0stzk/dotfiles#173]
Gh0stzk, good morning.

I wish you a merry Christmas and a great new year.
.
I would also like to thank you on the subject of the link below:
https://github.com/gh0stzk/dotfiles/issues/169
.
Your solution worked correctly. It cost!
.
Now I wanted to ask for your help again.
.
I wanted to see if there is a way to fix the position of the systray icons
Below is an example in print - Rice Cristina
![Captura de tela_2023-12-31_10-03-10](https://github.com/gh0stzk/dotfiles/assets/124194562/606d0e02-a548-471b-ab89-2cc053a95a0f)
.
This rice is Pamela
Print:
![Captura de tela_2023-12-31_10-08-09](https://github.com/gh0stzk/dotfiles/assets/124194562/6dbf7d23-728f-4399-8f2f-e64c8d7ef7f1)
.
Another thing.
Which file do I change to adjust the screen size of "helpKey -- alt+f1"?
.
Thanks!

**Comment by gh0stzk** [comment]
Hello, great new year for you too!

Yes the stalonetray is the only thing i can't adjust in percentage for all rices unfortunately.

But yes you can adjust for your resolution.

In the file Theme.sh

![Shot-2023-12-31-120724](https://github.com/gh0stzk/dotfiles/assets/67278339/25e2e80b-9442-4d0d-871d-2c2e01724d72)

The line with geometry word, yo need to edit the 998+54 vaues

**Comment by dbmb91** [comment]
Gh0stzk, good morning.

Being able to adjust the systray position... thanks.
.
Now can you help me with this question here?
Print:
![image](https://github.com/gh0stzk/dotfiles/assets/124194562/8d1f4c11-1864-4434-b8cf-ba5e78ef152c)
.
I want to change the size of this help-Key screen, but I can't.
I have already changed the values, but to no avail. :)
.
Hmm... one other thing
.

When I try to put a mouse theme in another rice... it is not updating the "index.theme" which is in the ".icons" folder
.
![image](https://github.com/gh0stzk/dotfiles/assets/124194562/0fe7ed59-1c96-4a98-a61f-4c335cc69e81)

.

I'm using this solution you gave me:
.
.
apply_appearance() {
	XFILE="~/.config/bspwm/xsettingsd"

	# apply gtk theme, icons, cursor & fonts
		sed -i -e "s|Net/ThemeName .*|Net/ThemeName \"Dracula\"|g" ${XFILE}
		sed -i -e "s|Net/IconThemeName .*|Net/IconThemeName \"Luv-Folders-Dark\"|g" ${XFILE}
		sed -i -e "s|Gtk/CursorThemeName .*|Gtk/CursorThemeName \"Bibata-Modern-Ice\"|g" ${XFILE}
}
.
.
It changes the mouse icon when I place it... but there are some apps that keep the theme that is assigned in "index.theme". Is there any way I can update the "index.theme" with the function above?

**Comment by gh0stzk** [comment]
Hello, about the rofi menu for the keybindings, i try it to change but i wasn't able to change the size, i nees to take a deeper search why is not changing.

About the icon theme, where is your theme located? in ~/.icons or /usr/share/icons??? 
But if it changes for some apps, means it works like suppose to work, but i think is more about specific app problem, somo apps are old or to new "gtk 4" , and is an app related issue.

**Comment by dbmb91** [comment]
OK, no problem.
I will test the solution above.
Thanks for your help!
And sorry for the inconvenience :)

**Comment by dbmb91** [comment]
Gh0stzk, good night.
Sorry for the delay in getting back.
.
I tried to implement the code you gave me above, but I couldn't.
.
I took a print to show the mouse pointer, but it doesn't appear in the print. hhAHA
print:
.
![Shot-2024-01-05-104001](https://github.com/gh0stzk/dotfiles/assets/124194562/412272d5-e833-4753-ba13-7c0850524539)
.
In the screenshot above there is a function that changes the color of the theme, icons and mouse.
.
I add the mouse theme "ArcStarry-cursors" and as soon as I update bspwm. The mouse theme is changed.
But for example... if I open Google Chrome and hover the mouse pointer over the Google screen. The theme that appears is the "Bibata-Modern-Ice" theme, which is the theme in the "index.theme" file
It is located at:
$HOME/.icons
.
Print:
![Shot-2024-01-05-104036](https://github.com/gh0stzk/dotfiles/assets/124194562/f4fa1558-b197-4a5b-b967-83d807251637)

Obrigado.
